categories/avideo-exploit.md

```diff
@@ -0,0 +1,10 @@
+## YouPHPTube 7.4  Remote Code Execution 
```

Comment:
Can we rename this into the CVE that this exploits and add the application and vendor details as "tags" ?